index/visualisation.py

```diff
@@ -7,14 +7,15 @@
 import matplotlib.pyplot as plt
 from sklearn.manifold import TSNE
 import plotly.graph_objects as go
+import plotly.express as px
 
 from index.conf import COLORS_AD, COLORS_PD
 from index.mapping import MappingTable
 
 
 class PlotSide(Enum):
-    LEFT = 1,
-    RIGHT = 2,
+    LEFT = (1,)
```

Comment:
Why did you add brackets here?

Reply:
It's the Black Formatter again, I will fix it.